packages/jobs/package.json

```diff
@@ -20,10 +20,10 @@
         "@octokit/plugin-retry": "^6.0.0",
         "@octokit/plugin-throttling": "^7.0.0",
         "@octokit/rest": "^20.0.1",
-        "@temporalio/activity": "^1.5.2",
-        "@temporalio/client": "^1.5.2",
-        "@temporalio/worker": "^1.5.2",
-        "@temporalio/workflow": "^1.5.2",
+        "@temporalio/activity": "^1.9.1",
+        "@temporalio/client": "^1.9.1",
+        "@temporalio/worker": "^1.9.1",
+        "@temporalio/workflow": "^1.9.1",
```

Comment:
Can you also upgrade shared and server?

Reply:
Updated for shared and the server wasn't using anymore so removed there. Thanks